server/app/acs_enterprise/lambdas/scheduler_lambda.py

```
@@ -105,38 +115,243 @@ async def _track_timed_out_calls(self, db: AsyncSession):
+                    f"✅ Reconciliation complete: removed {removed_count}/{len(stuck_calls)} stuck calls"
+                )
+
+            else:
+                # Perfect sync
+                logger.debug(
+                    f"✅ Counter in perfect sync ({len(dynamo_call_ids)} active calls, "
+                    f"DynamoDB IDs match PostgreSQL IDs exactly)"
+                )
+
+        except Exception as e:
+            logger.error(f"❌ Error reconciling concurrency counter: {str(e)}", exc_info=True)
```

Ruff (0.15.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)